### DISCUSSION TITLE
big js 404

### DISCUSSION THREAD

**Discussion by The27149** [community/community#120818] [Programming Help]
### Select Topic Area

Question

### Body

i can start my project  in the local host, but it's failed in githubpages. it looks like some js files so big that download failed. then i check these files in my repository. i even can't view them.  github remind me view raw at https://raw.githubusercontent.com/.  so how should i solve this problem. my english is not good, i hope some kind person understand and help me. thank you

**Comment by LiteBrite82** [comment]
@The27149 Can you provide a link to the repository?

**Reply by The27149** [reply]
of course, https://github.com/The27149/site.git. the big file at mathGraphics/cocos-js/_virtual_cc-f08d7e60.js. 
the githubPage address is https://the27149.github.io/site/，thank you once more

**Reply by LiteBrite82** [reply]
I couldn't identify the issue from a GH Pages perspective. It may be a coding issue; Im gonna send this over to Programming Help; maybe we can get some more insight from the community there?

**Reply by LiteBrite82** [reply]
as far as the specific file you pointed out; it is fine, it is just too big to view in the browser - it shouldn't affect functionality of the application on Pages

**Reply by The27149** [reply]
![image](https://github.com/community/community/assets/23302000/0bd02ce2-a13d-4361-95e8-d7e73f3e339e)
![image](https://github.com/community/community/assets/23302000/490586c5-113c-4c73-a0ff-873c78ba7d27)
as you see, it's failed because can't download the big file. if you click link try to request it.(https://the27149.github.io/site/mathGraphics/cocos-js/_virtual_cc-f08d7e60.js), then return 404;
